pkg/cluster/resources.go

```diff
@@ -343,11 +343,17 @@ func (c *Cluster) deleteService(role PostgresRole) error {
 	return nil
 }
 
-func (c *Cluster) createEndpoint(role PostgresRole) (*v1.Endpoints, error) {
+func (c *Cluster) createEndpoint(role PostgresRole, sync bool) (*v1.Endpoints, error) {
```

Comment:
I always find unnamed boolean function parameters to be a bit unclear.  Instead of using a `bool` parameter here, could we use the cluster status from `c.GetStatus()` and compare it to `ClusterStatusCreating`?